recipes/untwine/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ex
+
+
+if [ "$target_platform" = "osx-arm64" ]; then
+  export CMAKE_OSX_ARCHITECTURES="arm64"
+fi
+
+rm -rf build && mkdir build && cd build
+cmake ${CMAKE_ARGS} -G Ninja \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_PREFIX_PATH=$PREFIX \
+  ..
+
+ninja
+ninja install
```

Comment:
Is there a `make check` or `ctest` that we can run here and in the `bld.bat`?

Reply:
Not at this time. If it builds the `untwine` utility and spits out the correct version number, it is successful.